app/components/post/post_container.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {connect} from 'react-redux';
+
+import {getTheme} from 'service/selectors/entities/preferences';
+import {getUser} from 'service/selectors/entities/users';
+
+import Post from './post';
+
+function mapStateToProps(state, ownProps) {
+    return {
+        ...ownProps,
+        user: getUser(state, ownProps.post.user_id),
+        post: ownProps.post,
```

Comment:
That's correct about the ownProps part, but I don't think this component should be dumb. There's stuff like the posting user, their status, and any display settings that the `post_list` won't care about, but the `post` will that we don't want to need to pass down.